src/jit/hwintrinsicxarch.cpp

```diff
@@ -540,27 +540,28 @@ GenTree* Compiler::addRangeCheckIfNeeded(NamedIntrinsic intrinsic, GenTree* last
 // Arguments:
 //    isa - Instruction set
 // Return Value:
-//    true - all the hardware intrinsics of "isa" are implemented in RyuJIT.
+//    true - all the hardware intrinsics of "isa" exposed in CoreFX
+//    System.Runtime.Intrinsics.Experimental assembly are implemented in RyuJIT.
```

Comment:
I do not think we need the new comment. JIT implementation is not related to corefx surface and the `S.R.Intrinsic.Experimental` package is a temporary solution.

Reply:
> `S.R.Intrinsic.Experimental` package is a temporary solution

Agree, however, function logic has changed and old description is wrong. Perhaps I will add TODO-XARCH-Cq to describe it and remember to update function and comment in future. I would expect that once we have all ISAs implemented we will not need this check and function anymore.

Reply:
>  we will not need this check and function anymore.

Not really, I think that we always need this logic for future work (e.g., AVX-512) even though the current ISA support gets completed.

Reply:
> I think that we always need this logic for future work (e.g., AVX-512)

Yes, you are right, but only under the condition that we will follow implementation pattern where APIs are exposed first and implemented later.